dev/fattest.simplicity/bnd.bnd

```diff
@@ -92,3 +92,7 @@ test.project: true
 	org.jboss.arquillian.core:arquillian-core-api;version=1.3.0.Final,\
 	org.jboss.arquillian.test:arquillian-test-spi;version=1.3.0.Final,\
 	org.testcontainers:testcontainers;version=1.14.0
+	
+#	,\
+#	org.apache.sshd:sshd-core;version=2.5.1
```

Comment:
This change should be removed if not absolutely required

Reply:
Good catch. Originally I was going to add some utils for some remote interactions with the KDC, but instead put it in my FAT bucket.